analyzer/codechecker_analyzer/analyzers/infer/descriptions.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Please include the fb infer original license file
https://github.com/facebook/infer/blob/main/LICENSE
with the additional description that it is valid for the descriptions.json file